woocommerce/woocommerce
# Update Customize Your Store introduction page

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR is part of the Customize Your Store task that is currently under development. This PR adds visual components to the introduction page.

Closes #39528.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the 'customize-store' feature flag using the WooCommerce Admin Test Helper
2. At the WooCommerce Home screen, click on the Customize your store task.
3. The page should (mostly) match the screenshot below:

![Image](https://user-images.githubusercontent.com/9312929/257667942-173a603b-db83-4e19-b371-0a87c667effb.png)
